tests/test_rejson.py

```diff
@@ -82,6 +89,12 @@ def testNumMultByShouldSucceed(self):
         self.assertEqual(5, rj.jsonnummultby('num', Path.rootPath(), 2.5))
         self.assertEqual(2.5, rj.jsonnummultby('num', Path.rootPath(), 0.5))
 
+    def testToggleShouldSucceed(self):
+        "Test JSONToggle"
+
+        rj.jsonset('bool', Path.rootPath(), False)
+        self.assertTrue(rj.jsontoggle('bool', Path.rootPath()))
```

Comment:
I suggest to test the toggle is actually toggling
```suggestion
        rj.jsonset('bool', Path.rootPath(), False)
        self.assertTrue(rj.jsontoggle('bool', Path.rootPath()))        
        self.assertFalse(rj.jsontoggle('bool', Path.rootPath()))
```

Reply:
I suggest to test also an erroneous path, e.g., does not lead to a bool